packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/modelservice/ModelCompositionParams.java

```diff
@@ -0,0 +1,17 @@
+package software.uncharted.terarium.hmiserver.models.modelservice;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+import software.uncharted.terarium.hmiserver.models.modelservice.PetriNet;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+@Data
+@Accessors(chain = true)
+public class ModelCompositionParams implements Serializable {
+    private PetriNet modelA;
+    private PetriNet modelB;
+    private List<Map<String, String>> statesToMerge;
+}
```

Comment:
you should add a new line at the end of this file

Reply:
Why?

Reply:
because that's part of our linting configuration